src/components/attendee/VideoComms/VideoCommsParticipant.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useCallback, useState } from "react";
+
+import { UserId } from "types/id";
+
+import { useProfileById } from "hooks/user/useProfileById";
+
+import { UserAvatar } from "components/atoms/UserAvatar";
+
+import { AudioTrackPlayer } from "./internal/AudioTrackPlayer";
+import { VideoCommsControls } from "./internal/VideoCommsControls";
+import { useVideoComms } from "./hooks";
+import { Participant, VideoSource, VideoTrack } from "./types";
+import { VideoTrackDisplay } from "./VideoTrackDisplay";
+
+import styles from "./scss/VideoCommsParticipant.module.scss";
+
+interface VideoCommsParticipantProps {
+  participant: Participant;
+  isLocal?: boolean;
+  /**
+   * A function that will generate controls on a per-video-track basis. Useful for
+   * adding things like "share" buttons or similar on each track. The Video
+   * Huddle components makes use of this.
+   */
+  videoTrackControls?: (track: VideoTrack) => JSX.Element;
+}
+
+export const VideoCommsParticipant: React.FC<VideoCommsParticipantProps> = ({
+  participant,
+  isLocal,
+  videoTrackControls,
+}) => {
+  const {
+    startAudio,
+    stopAudio,
+    startVideo,
+    stopVideo,
+    isTransmittingAudio,
+    isTransmittingVideo,
+  } = useVideoComms();
+
+  const { profile, isLoading } = useProfileById({
+    userId: participant.sparkleId as UserId,
+  });
+
+  const hasActiveVideoStream = participant.videoTracks.some(
+    (t) => t.sourceType === VideoSource.Webcam && t.enabled
+  );
+
+  // These muted controls are only for muting the playback of remote participants
+  // Local participant audio is controlled via useVideoComms
```

Comment:
The playback of remote participants is basically a render time issue rather than something that needs to interact with Twilio (at least, with how we do it right now).

I think keeping it out of `useVideoComms` keeps that API cleaner, for now.

Based on conversations about other features we're wanting to build I suspect this API will change anyway.